devops/kube/monolith-manifest.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata: 
+  name: forge-frontend-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: forge-frontend-backend
+  template:
+    metadata:
+      labels:
+        app: forge-frontend-backend
+    spec:
+      containers:
+      - name: forge-frontend
+        image: dajpearl/forge-frontend:latest
+        resources:
+          requests:
+            memory: "500Mi"
+            cpu: "100m"
+          limits:
+            memory: "1Gi"
+            cpu: "200m"
+      - name: forge-backend
+        image: dajpearl/forge-backend:latest
+        resources:
+          requests:
+            memory: "500Mi"
+            cpu: "100m"
+          limits:
+            memory: "1.5Gi"
+            cpu: "200m"
+        ports:
+        - containerPort: 3000
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: frontend-backend-service
+spec:
+  type: LoadBalancer
+  selector:
+    app: forge-frontend-backend
+  ports:
+  - port: 3000
+    name: frontend
+    protocol: TCP
+  - port: 8081
+    name: backend
+    protocol: TCP
+
+
```

Comment:
random blank lines in this file; i wouldn't reject it over just this, but it does show sloppiness